internal/app/machined/pkg/controllers/k8s/control_plane.go

```diff
@@ -145,7 +140,29 @@ func NewControlPlaneAuthorizationController() *ControlPlaneAuthorizationControll
 					})
 				}
 
-				res.TypedSpec().Config = slices.Concat(v1alpha1.APIServerDefaultAuthorizationConfigAuthorizers, authorizers)
+				if !slices.ContainsFunc(authorizers, func(a k8s.AuthorizationAuthorizersSpec) bool {
```

Comment:
I'm confused why we add them to the tail of the list now, while previously they were at the head? Should we keep it at the head?

Reply:
if we keep at head, does that mean user provided always comes at the end, which is not what we want

Reply:
oh I understand now, keep at head if either `Node` or `RBAC` is not specified by the user, that makes sense